cassandra-diagnostics-core/COREMODULES.md

```diff
@@ -154,3 +154,24 @@ Default reporting interval is set to 10s in order to detect these short outages.
     - io.smartcat.cassandra.diagnostics.reporter.LogReporter
 ```
 
+## Hiccup module
+
+Module based on [jHiccup](https://github.com/giltene/jHiccup) that logs and reports platform hiccups including JVM stalls. Default reporting period is 5 seconds and reporter values and percentiles from 90 to 100 and Mean and Max values.
+
+#### Configuration
+
+```
+- module: io.smartcat.cassandra.diagnostics.module.hiccup.HiccupModule
+  options:
+    resolutionInMs: 1.0d #optional
+    startDelayInMs: 30000 #optional
+    allocateObjects: false #optional
+    lowestTrackableValue: 1000L * 20L #optional
+    highestTrackableValue: 3600 * 1000L * 1000L * 1000L #optional
```

Comment:
I think that hiccup is an advanced module and I would keep these but I agree that a better explanation should be put in readme.